cmd/gofish/ensure_unix.go

```diff
@@ -23,14 +36,20 @@ func ensureDirectories(dirs []string) error {
 	if err != nil {
 		return err
 	}
+
 	// strip the newline character from the end
 	curGroup := strings.TrimSuffix(string(groupOutput), "\n")
 
 	fmt.Printf("The following new directories will be created:\n")
 	fmt.Println(strings.Join(dirs, "\n"))
+	var cmd *exec.Cmd
 	for _, dir := range dirs {
 		if fi, err := os.Stat(dir); err != nil {
-			cmd := exec.Command("sudo", "mkdir", dir)
+			if isRoot {
```

Comment:
meh... Might be better just to call `os.MkdirAll` instead and rely on the user to call `sudo gofish init`. Or better yet, let the install script [create the appropriate directories](https://github.com/fishworks/gofish/blob/a9b4042ab53b6850c4fa6617c6cc579fd079be4d/scripts/install.sh#L103) leaving `gofish init` to fetch the default repository.

That way we don't have two codepaths requesting sudo permissions vs. regular permissions.

Reply:
Mmm, not sure if relying on the install script is the way.  This is touched upon https://github.com/fishworks/gofish/issues/150 

If we don't have the second codepath, then we'll need to error out on init if correct permissions aren't detected.  This is where we may want to have a `--local` flag or something.  Which would force a local install, where you may not need sudo.

This would also work for Docker, where sudo may not be available, and you might be running as a non-root user.

Reply:
Unfortunately I don't have time for that rewrite right now.  Will have to table it for a while.